cmd/controller/ctrl.go

```diff
@@ -96,7 +106,7 @@ func createDR(dr apis.Disk, c *Controller) {
 		 * one node to another in a cluster. So here we just have to change the
 		 * ownership of this disk resource to the current node.
 		 */
-		glog.Info("create failed, trying to update the disk resource : ", err)
+		glog.Info("Create failed, trying to update the disk resource : ", err)
```

Comment:
Should this be an Info or an Err?

Reply:
This log is a info

Reply:
Why should an error be put up as an info? Should this be a Warning?
However, you can ignore this comment if you strongly believe that this should be a Info

Reply:
We are using info as this is not exactly a error case we just put an info for debug using log.

Reply:
changed to warning.